app/controllers/report_controller.rb

```diff
@@ -58,7 +58,36 @@ def add_record
   end
 
   def raw_data
-    render json: Report.all
+    reports = Report.order(:max)
+    response = {}.tap do |res|
+      res[:header] = {}.tap do |h|
+        h[:user] = 'User'
+        h[:platform] = 'Platform'
+        h[:location] = 'Location'
+        h[:core_number] = 'NO. cores'
+        h[:ram] = 'RAM(GB)'
+        h[:main] = 'Equivalent CO2 emissions per core at full load (kgCO2eq/h)'
```

Comment:
You're not using this field I think?

Reply:
I think it is used at line 29 in `app/javascript/report/index.js`.

Reply:
Looking at it, I don't think that line in `index.js` is doing anything, the header text doesn't match `h[:main]` :shrug:

Reply:
It is used for rendering this field
![image](https://github.com/openflighthpc/carbon-leaderboard/assets/98431935/94703fef-970a-4b38-afea-dbc7cc9ea007)

Reply:
It's not that important to change, but in case it causes later confusion, I think that field is being determined in line 12 of `app/views/report/index.html.erb`. The units are different

Reply:
Ahh I see...Forgot to remove the one in the erb. Updated now

Reply:
And the selector in the javascript was incorrect. I was blind🤡